### PR TITLE
small addition from mid level focus group to cover engineers working …

### PR DESCRIPTION
**Pull request by mark-barnes**
…on proprietary systems (not using open source etc)

### CODE REVIEW

**Comment by alicebartlett** [comment]
This is a getting a bit wordy. 
Reuses existing code
Examples:
- Uses a node package in project
- Integrates with proprietary software packages
- Uses Origami

Rhys and I aren't entirely clear what "Proprietary software packages" means, or what an example of that would be. As this is an example on the competency though, that is not necessarily important as I assume the target audience for this change will know what it means.

**Comment by rowanmanning** [comment]
Me Rhys and Mark agree that Alice's suggestion is good. Will update this soon

**Comment by rowanmanning** [comment]
Thanks mark!
